fix: Traditional mode clustering info (#1206)
Kong/developer.konghq.com: app/gateway/traditional-mode.md

> _Figure 1: In a traditional deployment, all {{site.base_gateway}} nodes connect to the database. 
Each node manages its own configuration._

## About {{site.base_gateway}} clusters

> A {{site.base_gateway}} cluster does not load balance client traffic across Data Plane nodes out-of-the-box. You still need a
> [load-balancer](/gateway/load-balancing/) in front of your Data Plane nodes to distribute your traffic. Instead,
> a {{site.base_gateway}} cluster means that those nodes will share the same configuration.

Reviewer: ```suggestion
Data Plane nodes in the same {{site.base_gateway}} cluster share the same configuration.

To load balance traffic across Data Plane nodes in a cluster, configure a [load balancer](/gateway/load-balancing/) in front of your nodes to distribute the traffic. 
```
This feels like a more natural order to these facts.
Not sure that you need to say "gateway doesn't load balance traffic out of the box"; it's generally better to phrase it in a way of what you _can_ do, ie in this case "hey look, you can load balance across nodes, go configure it".

For performance reasons, {{site.base_gateway}} avoids database connections when proxying
requests, and caches the contents of your database in memory. The cached
[entities](/gateway/entities/) include Gateway Services, Routes, Consumers, plugins, credentials, and so on. Since those

> values are in memory, any change made via the Admin API of one of the nodes

Reviewer: ```suggestion
values are stored in memory, any change made via the Admin API of one of the nodes
```
